runtime/flang/getvolinfo3f.c

```diff
@@ -8,27 +8,25 @@
 /*
  *	getvolinfo3f.c - DFWIN GetVolumeInformation subprogram
  *	This is just the interface to the actual system routine from
- *	the dwfin module routine.  This routine is called as an 'alias'
+ *	the dfwin module routine.  This routine is called as an 'alias'
  *	when the module is called; this routine makes sure that the
  *	output character arguments are blank-filled.
  */
 
 #include <string.h>
 
-typedef char *LPSTR;
-typedef int DWORD;
+#if defined (_WIN64)
+#ifndef WIN32_LEAN_AND_MEAN
+#define WIN32_LEAN_AND_MEAN
+#endif
```

Comment:
Would `#define WIN32_LEAN_AND_MEAN` cause a warning or error if the macro is already defined? Is the `#ifndef` necessary?

Reply:
Yes, that could be removed.